layout.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I like the idea of storing these in a file.
I'll make sure to load them from here in the `generate.py` script.